credentials/src/test/kotlin/web5/sdk/credentials/PresentationExchangeTest.kt

```diff
@@ -69,8 +70,8 @@ class PresentationExchangeTest {
       )
       val vc = VerifiableCredential.create(
         type = "StreetCred",
-        issuer = issuerDid.uri,
-        subject = holderDid.uri,
+        issuer = issuerDid.did.uri,
```

Comment:
just fyi in ts it's `issuerDid.uri`
![Uploading image.png…]()

Reply:
```
export class BearerDid {
  /** {@inheritDoc Did#uri} */
  uri: string;
  ...
  }

Reply:
ooooooh tank u! i missed that bit 🙏  added `uri` to BearerDid and changed all the places where i'm calling `bearerdid.did.uri` to `bearerdid.uri`